.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: go
+go:
+  - "1.10.x"
+sudo: true
+
+before_install:
+  - sudo add-apt-repository -y ppa:maarten-fonville/protobuf
+  - sudo apt-get -qq update
+  - sudo apt-get install -qq protobuf-compiler
+  - go get github.com/golang/mock/mockgen
+           github.com/golang/protobuf/protoc-gen-go
+           github.com/dominikznidar/govendor
+           github.com/go-swagger/go-swagger/cmd/swagger
+  - make
+
+script:
+  - go test -short ./...
```

Comment:
Why not `make test`?

Reply:
To avoid 6 minutes of `storage/encrypted_bolt` tests.

Should I change `make test` to run short test?

Reply:
yes